en/docs/administration/customize-the-developer-portal.md

```diff
@@ -0,0 +1,48 @@
+# Customize the Developer Portal
+
+You can share APIs internally and externally through the Choreo Developer Portal. You can change the look and feel of your Developer Portal by changing the theme to match your brand. Doing so will help you give a better developer experience to your users.
+
+To configure the Developer Portal theme, follow the steps given below:
+
+!!! Note
+    - To change the Developer Portal theme for an organization, you need to be an admin user of that organization.<br/><br/>
+    - If you want to remove any theme change you make, you need to reset to the default theme. Choreo does not support undoing a specific change and reverting to the previous theme.
+
+1. Sign in to the Choreo Console at [https://console.choreo.dev/](https://console.choreo.dev/) using a Google, GitHub, or Microsoft account.
+
+2. If you are a new user, create an organization. Enter a unique organization name. For example, "Stark Industries".
+
+    ![Create an organization in Choreo](../assets/img/references/enterprise-login/create-choreo-organization.png){.cInlineImage-small}
+
+3. In the left pane, click **Settings**.
+
+4. In the **Organization** tab, click **Devportal Theme**.
+
+    ![Access Devportal theme](../assets/img/administration/devportal-theme/access-devportal-theme.png)
+
+    Once you access the theme, you can customize the Home page, color theme, font, header and footer, logos, etc., by expanding the **General**, **Home Page**, **Color Palette**, **Typography**, and **Footer Links** sections.
+
+5. Make a change to the theme. For example, let's change the title on the home page and the color theme. 
```

Comment:
let's change the title on the home page, and it's color theme

Reply:
It is not only the color theme of the home page that we are changing isn't it?